### DISCUSSION TITLE
Dynamic Dialog

### DISCUSSION THREAD

**Discussion by terrence-newton** [primefaces/community#331] [PrimeNG] [answered]
Is there any way to automatically maximize this, or any workarounds? I've tried setting the height and width to maximum, but that still leaves some space around the edges.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately, dynamicdialog does not offer such a feature. However, as an idea, you may somehow trigger the maximize function after the dialog displayed by getting access to its reference.
